packages/babel-plugin-apply-mdx-type-props/index.js

```diff
@@ -24,6 +24,15 @@ class BabelPluginApplyMdxTypeProp {
                   t.stringLiteral(jsxName)
                 )
               )
+
+              const {namespacedImports} = state.opts
+              const isNamespaceImport =
+                namespacedImports &&
+                namespacedImports.find(s => s.includes(jsxName))
+
+              if (isNamespaceImport) {
+                path.node.name.name += '.default'
```

Comment:
What if it's a commonjs module?

Reply:
This will still work. I tested this using parcel example.

> a component exported using CommonJS
```jsx
// Counter.js
import React from 'react'

const Counter = () => {
  return <div style={{fontSize: 40}}>Not a counter</div>
}

module.exports = Counter
```

> mdx file importing it as a namespaced import

```mdx
import * as Counter from './Counter'

# Hello, world!

<Counter />
```

> Used parcel example to test this.

<img width="502" alt="Screenshot 2020-05-25 at 12 08 35 PM" src="https://user-images.githubusercontent.com/6652823/82785565-79854100-9e80-11ea-883d-d7c96e6c4f5e.png">

Reply:
This will still work if the export was done on a commonjs module.